js/modules/types/attachment.js

```diff
@@ -107,3 +109,62 @@ exports.removeSchemaVersion = (attachment) => {
   delete attachmentWithoutSchemaVersion.schemaVersion;
   return attachmentWithoutSchemaVersion;
 };
+
+exports.migrateDataToFileSystem = migrateDataToFileSystem;
+
+//      hasData :: Attachment -> Boolean
+exports.hasData = attachment =>
+  attachment.data instanceof ArrayBuffer || ArrayBuffer.isView(attachment.data);
```

Comment:
One example where `module.exports` would make the function longer. But not a huge deal 😄 

BTW, this definition of `data` is what `window.crypto.subtle` expects for encryption.